src/components/Form.js

```diff
@@ -0,0 +1,12 @@
+import React from 'react';
+import './Form.scss';
+
+function Form() {
+    return (
+        <form className="Form">
```

Comment:
The form does not need a form class because it's the only form component on the page.